test/test_util/react_util.dart

```diff
@@ -156,8 +156,8 @@ Element renderAndGetDom(dynamic component, {bool addToRenderedQueue: true}) {
   return findDomNode(render(component, addToRenderedQueue: addToRenderedQueue));
 }
 
-/// Renders a React component or builder into a detached node and returns the associtated Dart component.
-react.Component renderAndGetComponent(dynamic component) => getDartComponent(render(component));
+/// Renders a React component or builder into a detached node and returns the associated Dart component.
+UiComponent renderAndGetComponent(dynamic component) => getDartComponent(render(component)) as UiComponent;
```

Comment:
@greglittlefield-wf I changed it because on line 335 of `common_component_tests.dart`, `consumedProps` is not available on `Component`.

Should I just do a cast to `UiComponent` in the test?